src/System.Linq.Expressions/tests/ExceptionHandling/ExceptionHandlingExpressions.cs

```diff
@@ -1073,10 +1073,8 @@ public void TryCatchThrowingWithinFilterCompiled()
         }
 
         private bool MethodWithManyArguments(
-            int x, int y, int z,
-            int α, int β, int γ, int δ,
-            int klaatu, int barada, int nikto,
-            int anáil, int nathrach, int ortha, int bháis, int @is, int beatha, int @do, int chéal, int déanaimh,
+            int a, int b, int c, int d, int e, int f, int g, int h, int i, int j,
```

Comment:
I think this one might have been intentional

Reply:
For what purpose? It's part of an exception handling test.